src/docs/user/hcatalog.txt

```diff
@@ -61,6 +61,10 @@ The presence of the +--hcatalog-table+ option signifies that the import
 or export job is done using HCatalog tables, and it is a required option for
 HCatalog jobs.
 
++--hcatalog-external-table+::
+Set this flag if you need to create external Hive table for example to store
```

Comment:
Instead of ORC, I'd mention non-transactional tables, since that's stopping you from creating HCat table without this patch.